submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/ProjectResource.kt

```diff
@@ -25,5 +25,5 @@ class ProjectResource(private val projectService: ProjectService) {
     @ApiImplicitParam(name = "X-Session-Token", value = "User authentication token", required = true)
     fun getUserProjects(
         @BioUser user: SecurityUser
-    ): List<Project> = projectService.getAllowedProjects(user, AccessType.ATTACH)
+    ): List<Collection> = projectService.getAllowedProjects(user, AccessType.ATTACH)
```

Comment:
We'll need to change here:

* The endpoint should be changed to `/collections`
* Class name: `CollectionResource`
* Variable name at line 21: `collectionService`
* Method name `getUserCollections`